### PR TITLE
Added the scroll to top with enhanced ui and fixed someone has putted the css directly on button which is bad habbit fixed that

### PR DESCRIPTION
**Pull request by tarunkumar2005**
# Pull Request Format
## Type of PR
- **Add X in the box to specify the improvement type.**
- [X] Bug fix
- [X ] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): ___________  

## Description
I have created the scroll to bottom button with enhanced ui and user experience looking very amazing and fixed some isses by inexperienced people directly applied styles to button it took me 3 min to find that someone has already applied the styles directly to the button tag which is not good.

## Screenshots / Videos (if applicable)
**Before:**

https://github.com/user-attachments/assets/c1b48029-cb37-4bf2-a108-3c03e2cbc00c

**After:**

https://github.com/user-attachments/assets/63d8059f-d642-4c30-892d-f156924695ae

## Checklist
- **Add X in the box to specify.**
- [X] I have performed a self-review of my code.
- [ X] I have tested the changes thoroughly before submitting this pull request.
- [ X] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [ X] I have commented my code, particularly in hard-to-understand areas.

Thank you for reviewing my pull request!

### CODE REVIEW

**Comment by tarunkumar2005** [comment]
@swaraj-das Check this and try to check on people who are doing like this applying styles directly on tag and so okay it is not good to do so. Merge it when you have checked this.

File: .github/workflows/auto-label-action.yml [hunk 1]
```diff
@@ -9,6 +9,7 @@ on:
 permissions:
   issues: write
   pull-requests: write
+  contents: read # Required to read repository contents
```
Comment (swaraj-das) [inline .github/workflows/auto-label-action.yml:12]: @AnitSarkar123 Check this.

**Review by AnitSarkar123 — APPROVED** [review]

**Comment by tarunkumar2005** [comment]
@swaraj-das and @AnitSarkar123 If you have reviewed then you can add labels and assign it to me and merge it.

**Comment by tarunkumar2005** [comment]
@swaraj-das and @AnitSarkar123 Please merge this PR if you have no issue i have by mistake added the new changes in the same PR which is not good can we talk over this please another issue i rasied and solved is merged into this PR only respond asp if you can

**Comment by tarunkumar2005** [comment]
issue #110 is also done in this only by mistake i pushed it and it is committed here now

**Comment by tarunkumar2005** [comment]
@swaraj-das and @AnitSarkar123 If you get time then you can tell me the conflicts are increasing with the the increase time so i will have to do more work in that case.

**Comment by AnitSarkar123** [comment]
varcel can't deployed it so we can't marge it at this time ...also make this pr again with an another branch because 2 issues solved in one you can't get the points of the tow issues

**Review by AnitSarkar123 — CHANGES_REQUESTED** [review]
deployment failed and rise another pr

**Comment by tarunkumar2005** [comment]
will create another PR for the scroll

**Pull request closed without merging**